src/app/actions.tsx

```diff
@@ -11,3 +15,32 @@ export async function createNewSection(formdata: FormData) {
   }
   await createSection(data)
 }
+
+export async function createNewThread(text: string, formdata: FormData) {
+  const content = formdata.get("content") as string
+
+  // Check if content is empty
+  if (!content.trim()) {
+    alert("Content cannot be empty.")
+    return // Exit the function early
+  }
+
+  if (content === "<p></p>") {
+    alert("Content cannot be empty.")
+    return
+  }
+
+  const topicId = formdata.get("topicId") as string
+  const sectionId = await getSectionId(topicId)
+
+  console.log(text)
```

Comment:
use logger.debug() instead of console.log()

```typescript
import {logger} from "@projectforum/lib/logger"
```

```suggestion
logger.debug(test)
```